helix-view/src/graphics.rs

```diff
@@ -95,31 +95,19 @@ pub struct Rect {
 }
 
 impl Rect {
-    /// Creates a new rect, with width and height limited to keep the area under max u16.
-    /// If clipped, aspect ratio will be preserved.
+    /// Creates a new rect, with width and height
     pub fn new(x: u16, y: u16, width: u16, height: u16) -> Rect {
```

Comment:
Does Rect need these fields to become `usize`s so that Rects can be drawn on very large terminal sizes?

Reply:
I don't think so. 65 thousand characters as a width or height is already insanely large. Also `crossterm` only works with `u16` for `x/y` positions.
Looking at `termwiz` ([which I understand we might switch to eventually](https://github.com/helix-editor/helix/issues/1472#issuecomment-1008644709)), they do use `usize`.
I just made sure that it was fine if the `area` which equals `width * height` is bigger than `2^16`, which could plausibly happen with a terminal around `256 by 256`.